src/validators/matchesValue.js

```diff
@@ -0,0 +1,13 @@
+import createValidator from '../createValidator';
+
+export default function matchesValue(matchKey, matchLabel) {
+  return createValidator(
+    message => (value, allValues) => {
+      if (value !== allValues[matchKey]) {
```

Comment:
Should we check for the existence of `allValues` and return the error message if it's falsey too?